parser_test.go

```diff
@@ -2448,6 +2448,54 @@ func (s *testParserSuite) TestPriority(c *C) {
 	c.Assert(sel.SelectStmtOpts.Priority, Equals, mysql.HighPriority)
 }
 
+func (s *testParserSuite) TestSQLBigResult(c *C) {
```

Comment:
Let's merge all three into the same test function.

Also, I suggest we use the `s.RunTest(c, table)` method for testing instead of the `TestSQLNoCache` approach (see `TestPrepare`), which is used to test `Restore()`.

There is no need to verify the value of `sel.SelectStmtOpts.SQLBigResult` since it is only used for `Restore()` and otherwise ignored. 

BTW please add a test case for 

```sql
select sql_small_result sql_big_result sql_buffer_result 1;
```

Reply:
ok, I will modify